system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -13,34 +13,36 @@
 
 (use-fixtures :each
               (join-fixtures
-               [(ingest/reset-fixture {"provguid1" "PROV1" "provguid2" "PROV2"})
-                (subscription-util/grant-all-subscription-fixture {"provguid1" "PROV1"} 
-                                                                  [:read :update]
-                                                                  [:read :update])
-                (subscription-util/grant-all-subscription-fixture {"provguid1" "PROV2"}
+               [(ingest/reset-fixture
+                  {"provguid1" "PROV1" "provguid2" "PROV2" "provguid3" "PROV3"})
+                (subscription-util/grant-all-subscription-fixture
+                  {"provguid1" "PROV1" "provguid2" "PROV2"}
+                  [:read :update]
+                  [:read :update])
+                (subscription-util/grant-all-subscription-fixture {"provguid1" "PROV3"}
                                                                   [:read]
                                                                   [:read :update])]))
```

Comment:
Need to test special cases on a provider that doesn't impact the other tests. There is one other test that's using PROV2 before, but it succeeded because it was making use of the permission in the cache for PROV1.